### PR TITLE
update locarna recipe (1.8.12)

### PR DESCRIPTION
**Pull request by s-will**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by daler** [comment]
@s-will can you rebase and retry? #3080 fixed some issues with the build system.

**Comment by s-will** [comment]
Hi @daler, @bgruening,
the first problem is fixed -- thanks a lot! However, there is another one:
  Command failed: /bin/bash -x -e /opt/conda/conda-bld/work/locarna-1.8.12/conda_build.sh
after the package is built and tested, which seems related to the build system.

**Comment by daler** [comment]
It's kind of confusing but the actual error is up a bit higher. In this case it looks like a perl path issue: https://travis-ci.org/bioconda/bioconda-recipes/jobs/180639795#L994

You can take a look at these docs to get familiar with how to read the bioconda travis logs:
http://bioconda.github.io/bioconda-utils/troubleshooting.html#reading-bioconda-utils-logs-on-travis-ci

**Comment by s-will** [comment]
Hi @bgruening, could you please help. Shouldn't this work by now? The test works for osx but still fails for linux; I hope, I didn't mess this up -- anyway I am stuck.

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ env:
   global:
   - secure: WS9nwlZfrSgdU5bUw9ZXh5xD067PyUVq+UZnPazORgh5kgkTTX4EsNnXPtmUiQ9DOwVvBiCZzn6c7N+YPuUdEROOR/pBjtFA+w/APFk4ug1tcIzRH7z2we+fZaAnKK3SRqOhHkUYa8tjZFGr53RsaKaxKEc0SIqsHGbDupdBz9KC4raXyUQobcLxWbE6BQq2jhyJGqWxOrhTTrd3WCRKVTgjUlQmwimk6s9txfnEbEguE7UlRhlO0CSqgBj2oUvwtJQSkZWAFminUi2Jcmnduebu42iYX6IF3bYNLVxy8d56v2APGv6reipsD9m3l6URRtpgGygMF4EI9X6dlCjT2qMrwwuWQcZJncFkR584dL+MiPfA2RXamGQybX8HSlnrn60ZKzr5gmw4hetffUGwmOA2VlEW1Wcxcwcef+/8osWwnfLT4I3i736P71LARcBWjBaXKKCXCkCLegv0WB0V/JXyo6IfdLToB0ElAwNfoSlhK1ZtTejngMQewzh6HFcyJwbyPog8/WCL+/j+OohdLZYgiXuldeoLoTwM8YmrSV7twUg3rl2Uc9AWd6iKXC2n8t+2XXdSTwcyrLAEp5J2gZA+HPE/4R6/K1YLbILfMp+u2k8TpazSVX7oRwIcp5UJK6NGx9KDLnT5nJXFW2J5SuBdvQjOXGNSVxyAhBg2AAM=
   - SUBDAGS=1
-  - BIOCONDA_UTILS_TAG=master
+  - BIOCONDA_UTILS_TAG=reqs-updates
```
Comment (bgruening) [inline .travis.yml]: This should not be needed by now.
Are you sure you have rebased against latest master?

File: simulate-travis.py [hunk 2]
```diff
@@ -59,15 +59,15 @@
 HERE = os.path.abspath(os.path.dirname(__file__))
 
 
-def _remote_or_local(fn, remote=False):
```
Comment (bgruening) [inline simulate-travis.py]: This is also not needed anymore.

**Comment by s-will** [comment]
replaced by PR #3115

**Pull request closed without merging**